opm/grid/polyhedralgrid/geometry.hh

```diff
@@ -130,26 +135,24 @@ namespace Dune
     PolyhedralGridBasicGeometry ( ExtraData data, const EntitySeed& seed )
     : storage_( data, seed )
     {
-      GeometryType myType = type();
+      /*
```

Comment:
Is it important to keep the commented-out code?

I do not mind merging this when Jenkins is green.